src/main.cpp

```diff
@@ -5,11 +5,14 @@
 #include "commandlineparser.h"
 #include "leakdetector.h"
 
+#include "telemetry.h"
+
 int main(int argc, char* argv[]) {
 #ifdef MVPN_DEBUG
   LeakDetector leakDetector;
   Q_UNUSED(leakDetector);
 #endif
+  Telemetry::recordAppStartTimestamp();
```

Comment:
We can have two events here, "main screen started loading" / "main screen lof complete" instead of calculating the time ourselves. Then we record the first event here and the other even later and let Glean do the timestamp calculations. WDYT?

Reply:
My problem in general would rather be, can we record glean events before glean is initialised or what do you think about 2 events that have a "timestamp" extra ?

Reply:
Events have timestamps by default, so the extra is not necessary. And yes, we can record Glean events before init, they will just be sent after init. There is a limit for how many events can be recorded before init, but that is very high and will not be hit here.

Reply:
> And yes, we can record Glean events before init, they will just be sent after init. 

ie. during "Telemetry::recordAppStartTimestamp();" - Neither the QJs engine is created nor the android daemon started yet - so i mean glean code has not yet been loaded, which can store the event for later. That's why i'd use an extra timestamp.

Reply:
Sheesh, I see... Ok yeah I don't see how that would work in this case. Well, then a single event with the two timestamps as extras should suffice no? The Glean event timestamps are relative to the first event recorded on each app run, they are not absolute. So if we want to compare our two timestamps we would need to calculate them both ourselves. 

Ugh, that's messy for sure.